portfolio/src/main/java/com/google/Constants.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
```

Comment:
Could you place this file under `java/com/google/sps/servlets` and change its package to `com.google.sps.servlets`? Once that's done, you should be able to remove the import statement for it in `DataServlet.java` because it will be in the same package.

Reply:
Thanks! I got it to work.